numpy/_core/numeric.py

```diff
@@ -709,6 +709,8 @@ def correlate(a, v, mode='valid'):
     mode : {'valid', 'same', 'full'}, optional
         Refer to the `convolve` docstring.  Note that the default
         is 'valid', unlike `convolve`, which uses 'full'.
+    out : ndarray, optional
+        A location where the result is stored
```

Comment:
```suggestion
        A location where the result is stored.
```
Might prefer something along `Alternative output array in which to place the result.`  which is used elsewhere.  But this is fine... It seems that we don't have a uniform spelling here.